client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notifcation-settings/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { Button } from '@automattic/components';
+import { Modal, ToggleControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect, useMemo, useState } from 'react';
+import clockIcon from 'calypso/assets/images/jetpack/clock-icon.svg';
+import SelectDropdown from 'calypso/components/select-dropdown';
+import TokenField from 'calypso/components/token-field';
+import type { MonitorSettings } from '../../sites-overview/types';
+
+import './style.scss';
+
+type Duration = { label: string; time: number };
+
+interface Props {
+	site: { blog_id: number; url: string };
+	onClose: () => void;
+	settings: MonitorSettings | undefined;
+}
+
+export default function NotificationSettings( { onClose, site, settings }: Props ) {
+	const translate = useTranslate();
+
+	const [ enableMobileNotification, setEnablMobileNotification ] = useState< boolean >( false );
+	const [ enableEmailNotification, setEnableEmailNotification ] = useState< boolean >( false );
+	const [ selectedDuration, setSelectedDuration ] = useState< Duration | undefined >( undefined );
+	const [ addedEmailAddresses, setAddedEmailAddresses ] = useState< string[] | [] >( [] );
+
+	const mobileAppLink = 'https://jetpack.com/blog/jetpack-mobile-app/';
```

Comment:
@madebynoam - Is this link ok?

Reply:
Ooh, good question, we have a slightly newer page here that we could use:
https://jetpack.com/mobile/

Reply:
Sure, thanks :)

Reply:
Updated here - https://github.com/Automattic/wp-calypso/pull/71373/commits/5ba3e76f146eeb6a6fc20651f313953d74ba45e0